.github/workflows/publish.yaml

```
@@ -106,6 +115,11 @@ jobs:
           .venv/bin/python -m pip install --upgrade pip
           .venv/bin/python -m pip install uv
       
+      - name: Set Version
+        run: |
+          # Set version in pyproject.toml for setuptools-scm
+          echo "SETUPTOOLS_SCM_PRETEND_VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)